components/Grid.js

```diff
@@ -5,9 +5,9 @@
 
 import propTypes from '@styled-system/prop-types';
 import styled from 'styled-components';
-import { color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
+import { border, color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
 
-const boxProps = compose(space, color, layout, typography, flexbox, grid);
+const boxProps = compose(space, color, layout, typography, flexbox, grid, border);
```

Comment:
Each time we add a new prop there, we make the component slower. Can we keep this one about layout only, like it was in the lib we imported it from? We already have the `Container` component for cases where we need to add a border.

Reply:
I don't think the performance impact of this is relevant but I agree we should be consistent and keep it layout only.